packages/hooks/src/useWalletsList.ts

```diff
@@ -0,0 +1,119 @@
+import { useStore } from '@deriv/stores';
+
+type TWalletAccount = {
+    account_category?: 'trading' | 'wallet';
+    account_type?: string;
+    balance: number;
+    currency: string;
+    is_disabled: boolean;
+    is_virtual: boolean;
+    landing_company_shortcode: 'svg' | 'malta';
+    loginid: string;
+    icon: string;
+    icon_type: 'fiat' | 'crypto' | 'app';
+    name: string;
+};
+
+// TODO: Will be deleted and replaced by original function from @deriv/shared
+const getWalletCurrencyIcon = (currency: string, is_dark_mode_on: boolean, is_modal = false) => {
+    switch (currency) {
+        case 'demo':
+            if (is_modal) return 'IcWalletDerivDemoLight';
+            return is_dark_mode_on ? 'IcWalletDerivDemoDark' : 'IcWalletDerivDemoLight';
+        case 'USD':
+            return 'IcWalletCurrencyUsd';
+        case 'EUR':
+            return 'IcWalletCurrencyEur';
+        case 'AUD':
+            return 'IcWalletCurrencyAud';
+        case 'GBP':
+            return 'IcWalletCurrencyGbp';
+        case 'BTC':
+            return is_dark_mode_on ? 'IcWalletBitcoinDark' : 'IcWalletBitcoinLight';
+        case 'ETH':
+            return is_dark_mode_on ? 'IcWalletEtheriumDark' : 'IcWalletEtheriumLight';
+        case 'USDT':
+        case 'eUSDT':
+        case 'tUSDT':
+        case 'UST':
+            if (is_modal) {
+                return is_dark_mode_on ? 'IcWalletModalTetherDark' : 'IcWalletModalTetherLight';
+            }
+            return is_dark_mode_on ? 'IcWalletTetherDark' : 'IcWalletTetherLight';
+        case 'LTC':
+            return is_dark_mode_on ? 'IcWalletLiteCoinDark' : 'IcWalletLiteCoinLight';
+        case 'USDC':
+            return is_dark_mode_on ? 'IcWalletUsdCoinDark' : 'IcWalletUsdCoinLight';
+        default:
+            return 'Unknown';
+    }
+};
+
+// TODO: Will be deleted and replaced by original function from @deriv/shared
```

Comment:
@sergei-deriv why didn't you put it in the shared package instead of left a comment for change ?

Reply:
@mahdiyeh-deriv It's because hook package doesn't have @deriv/shared package as dependency in package.json. I've asked Farzin about this and he said to leave these function here and move them to @deriv/utils package when it will be released.